es6-promises/main.js

```diff
@@ -0,0 +1,11 @@
+import takeAChance from './take-a-chance.js';
+
+const user = takeAChance('Justin');
+
+user.then((value) => {
+  console.log(value);
+});
+
+user.catch((reason) => {
+  console.error(reason.message);
+});
```

Comment:
To properly handle errors, these need to be chained:
```
user.then(...).catch(...);
```
Usually you would skip the intermediate variable and just use:
```
takeAChance('Justin')
  .then(...)
  .catch(...);
```